index.js

```diff
@@ -3,13 +3,13 @@
 'use strict';
 
 const Breaker = require('circuit-fuses').breaker;
-const Request = require('request');
 const Hoek = require('@hapi/hoek');
 const Joi = require('joi');
 const Path = require('path');
 const Schema = require('screwdriver-data-schema');
 const Scm = require('screwdriver-scm-base');
 const logger = require('screwdriver-logger');
+const got = require('./lib/got');
```

Comment:
It would be a good idea to make an sd package wrapper for got.js, exposing methods like get, post, put, delete

Reply:
Created new library: https://github.com/screwdriver-cd/request